config/webpack/webpack.bundle.config.js

```diff
@@ -7,9 +7,9 @@ module.exports = {
   entry: helpers.root('.srctmp/index.ts'),
   output: {
     path: path.resolve(__dirname, '..', '..', 'dist', 'bundles'),
-    filename: 'core.umd.js',
+    filename: 'stache.umd.js',
```

Comment:
Yeah I think that's something to keep in mind as configurable when we migrate to builder.